instrumentation/spymemcached-2.12/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spymemcached/SpymemcachedTest.java

```diff
@@ -879,10 +896,12 @@ void decrException() {
                         .hasException(
                             new IllegalArgumentException("Key is too long (maxlen = 250)"))
                         .hasAttributesSatisfyingExactly(
-                            equalTo(
-                                maybeStable(DB_SYSTEM),
-                                DbIncubatingAttributes.DbSystemIncubatingValues.MEMCACHED),
-                            equalTo(maybeStable(DB_OPERATION), "decr"))));
+                            withErrorType(
+                                "java.lang.IllegalArgumentException",
+                                equalTo(
+                                    maybeStable(DB_SYSTEM),
+                                    DbIncubatingAttributes.DbSystemIncubatingValues.MEMCACHED),
+                                equalTo(maybeStable(DB_OPERATION), "decr")))));
```

Comment:
Another way this cold be approached is using `equalTo(ERROR_TYPE, SemconvStability.emitStableDatabaseSemconv() ? "java.lang.IllegalArgumentException"  : null)`